From cdcc387f6f1287da1edda418d746e6c2c772e5bd Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Thu, 4 May 2017 15:44:51 -0400
Subject: [PATCH] Ticket 49241 - Update man page and usage for db2bak.pl

Description:  The usage and man page should state thtthe backup directory
              is actually a symlink to the the server's backup directory.
              Otherwise it is misleading, and could eventaully lead to
              diskspace issues.

https://pagure.io/389-ds-base/issue/49241

Reviewed by: firstyear(Thanks!)

(cherry picked from commit 0804c43991fa29ef7bd946b3e5a37844e2b87da4)
---
 ldap/admin/src/scripts/db2bak.pl.in | 4 ++--
 man/man8/db2bak.pl.8                | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/ldap/admin/src/scripts/db2bak.pl.in b/ldap/admin/src/scripts/db2bak.pl.in
index c73caa1..73d4187 100644
--- a/ldap/admin/src/scripts/db2bak.pl.in
+++ b/ldap/admin/src/scripts/db2bak.pl.in
@@ -33,8 +33,8 @@ sub usage {
     print(STDERR "        -w -          - Prompt for Directory Manager's password\n");
     print(STDERR "        -Z serverID   - Server instance identifier\n");
     print(STDERR "        -j filename   - Read Directory Manager's password from file\n");
-    print(STDERR "        -A backupdir  - Backup directory (backupdir/ID-<date_time>)\n");
-    print(STDERR "        -a backupdir  - Backup directory\n");
+    print(STDERR "        -A backupdir  - Backup directory symlink(backupdir/ID-<date_time>)\n");
+    print(STDERR "        -a backupdir  - Backup directory symlink\n");
     print(STDERR "        -t dbtype     - Database type (default: ldbm database)\n");
     print(STDERR "        -P protocol   - STARTTLS, LDAPS, LDAPI, LDAP (default: uses most secure protocol available)\n");
     print(STDERR "        -h            - Display usage\n");
diff --git a/man/man8/db2bak.pl.8 b/man/man8/db2bak.pl.8
index a752885..c51ccae 100644
--- a/man/man8/db2bak.pl.8
+++ b/man/man8/db2bak.pl.8
@@ -47,7 +47,7 @@ The name of the file that contains the root DN password.
 The backend database type (default: ldbm database).
 .TP
 .B \fB\-a\fR \fIbackupdir\fR
-The directory where the backup should be stored.
+The directory where the backup should be stored.  This directory is a symbolic link to the actual backup files located under "nsslapd-bakdir" directory that is set in the "cn=config" entry.
 .TP
 .B \fB\-A\fR \fIbackupdir\fR
 This is similar to \fB-a\fR, except that a sub-directory of \fIbackupdir\fR will be created for the backup, and the name of the sub-directory will be a timestamp of the form \fIserver-instance-date_time\fR.
-- 
2.9.3

